swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -321,6 +321,24 @@ public SSDBComboBox() {
 		listItemFormat.setPattern(JDBCType.DATE, dateFormat);
 		listItemFormat.setSeparator(separator);
 	}
+	
```

Comment:
New constructor for SSDBComboBox that does not require the query. Query isn't executed until later and my screen  helper isolates setting and execution of queries into a single method for a screen so it was redundant to pass the query into the Constructor.

Reply:
Any reason to deprecate the constructor with the query?

Reply:
Thought about it, but nah. It doesn't hurt anything and if you're not using the form helper classes then it doesn't really matter where you set the query.